package.json

```diff
@@ -99,6 +98,7 @@
     "mml-react": "^0.4.7"
   },
   "peerDependencies": {
+    "@breezystack/lamejs": "^1.2.7",
```

Comment:
I don't think we should make this lib a peer dependency. `await import()` should be enough to instruct the bundler to create a new chunk with it

Reply:
I will remove it

Reply:
I mean, it should stay as it was - a regular dependency but bundled in a different chunk

Reply:
reverted to regular dependency. I have tested it with devDependency + peerDependency scenario and the resulting app bundle included lamejs in a separate file. That is why I thought devDep would be ok.